samples/worker/Makefile

```diff
@@ -10,4 +11,9 @@ distclean:
 	rm -f $(PROGS) *.o
 
 worker: worker.o
-	g++ $(CXXFLAGS) $(CPPFLAGS) $(LDFLAGS) -o worker worker.o
+	$(CXX) $(CXXFLAGS) $(CPPFLAGS) $(LDFLAGS) -o worker worker.o
+
+install: all
+	@echo "--- The example app $(PROGS) is not installed ---"
```

Comment:
Such kind of message will mislead the user, (s)he will decide that smth wrong happened during installation and it should be fixed somehow.